File: examples/file/save/node/saveFile.js
const { FileService } = require("micro-js-client/file");

// Save a file
async function saveFile() {
  let fileService = new FileService(process.env.MICRO_API_TOKEN);
  let rsp = await fileService.save({
    file: {
      content: "file content example",
      path: "/document/text-files/file.txt",
      project: "examples",
    },
  });
  console.log(rsp);
}

saveFile();

File: examples/go/file/save/saveFile.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/file"
)

// Save a file
func SaveFile() {
	fileService := file.NewFileService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := fileService.Save(&file.SaveRequest{
		File: &file.Record{
			Content: "file content example",
			Path:    "/document/text-files/file.txt",
			Project: "examples",
		},
	})
	fmt.Println(rsp, err)
}
